fabulaws/ubuntu/packages/postgres.py

```diff
@@ -211,14 +211,17 @@ def pg_set_sysctl_params(self, restart=True):
         if restart:
             self.pg_cmd("restart")
 
+    @uses_fabric
+    def pg_replace_pw_encryption(self, before="scram-sha-256", after="md5"):
+        files.sed(self.pg_hba, before=before, after=after, flags="i", use_sudo=True)
+
     @uses_fabric
     def pg_allow_from(self, ip_ranges, restart=True):
         """Allow external connections from the given IP range."""
-
         self.pg_set_str("listen_addresses", "*")
         files.uncomment(self.pg_hba, "local +replication", use_sudo=True)
         for ip_range in ip_ranges:
-            hostssl_line = "hostssl    all    all    %s    md5" % ip_range
+            hostssl_line = f"hostssl    all    all    {ip_range} {self.postgresql_settings.get('password_encryption', default='md5')}"
```

Comment:
The second parameter could be assigned to a variable first to shorten this line (and the similar one below).